### DISCUSSION TITLE
stlite -- streamlit compatibility table

### DISCUSSION THREAD

**Discussion by lukefullard** [whitphx/stlite#340] [Ideas]
Firstly, fantastic work on stlite, stlite sharing, and stlite desktop!
I find stlite sharing really helpful to troubleshoot issues on the streamlit forum. I also have plans to utilise stlite desktop at my workplace where people cannot have Python installed.

Anyway, my suggestion: Could we please add a comparison table on the stlite github page with which features have been tested and work with stlite? For example, multipage apps works, but I am unsure about other streamlit features, and particularly some of the more popular components such as streamlit aggrid and folium.
Perhaps it is more useful to say what does not work with stlite.

Anyway, just a thought. Thanks again.

**Comment by whitphx** [comment]
Thank you, I am so glad to hear that stlite can actually be helpful in real-world situations. Please let me know if you find some use cases or experience some more applications!

w.r.t. the feature table, it makes sense. However, maintaining it takes much cost (automated tests for each component have not been set up yet).

So, 
> Perhaps it is more useful to say what does not work with stlite.

I added something like this suggestion.
Please check https://github.com/whitphx/stlite#limitations
